final_prep/README.md

```diff
@@ -43,32 +43,33 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 - [ ] Personal, Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
+![Screen Shot 2022-04-23 at 2 35 09 PM](https://user-images.githubusercontent.com/102785515/164937101-61731d8f-dd1c-43ae-8b46-66c1d2b3470a.png)
+![Screen Shot 2022-04-23 at 2 35 16 PM](https://user-images.githubusercontent.com/102785515/164937017-77defb29-1b78-4e65-b3e5-dcc17d8038fc.png)
```

Comment:
This looks great! Love the meditation time, all the "human" times, AND it seems like you have a realistic amount of time dedicated to Turing.